docs/victory-zoom-container/docs.md

```diff
@@ -0,0 +1,130 @@
+# VictoryZoomContainer
+
+`VictoryZoomContainer` provides pan and zoom behavior for any Victory component that works with an
+x, y axis. To add panning and zooming behavior, add `VictoryZoomContainer` as the
+`containerComponent` of your chart.
+
+```jsx
+<VictoryChart containerComponent={<VictoryZoomContainer/>}>
+  <VictoryLine data={data} />
+  <VictoryBar data={moreData}/>
+</VictoryChart>
+```
+
+## Props
+
+`VictoryZoomContainer` uses a superset of props used by [VictoryContainer].
+
+
+### zoomDomain
+
+The optional `zoomDomain` prop describes the zoomed and panned state. This prop is an object that
```

Comment:
I'd leave out "and panned". I think zoomed will help people think about it correctly.